integr8ly/installation
# Imp(Inventory file): Issue 175 - Remove the hosts file from Git and replace with a template file which should be used to create the hosts file

## Motivation
Issue: https://github.com/integr8ly/installation/issues/175

## What
- Rename the host file to host.template
- Add step in the README
- Add the host file in the .gitigonre

## Why
- Make easier to work with and avoid commit changes on it by mistake.

## Verification Steps 
Check the steps in the README. See [here](https://github.com/integr8ly/installation/blob/43ea2bce8aa7cde932bee67a67f684a35893588d/README.md#2-create-the-inventory-hosts-file)

## Checklist:


## Progress
